pkgs/development/tools/build-managers/bazel/shebang-test.nix

```diff
@@ -0,0 +1,46 @@
+{
+  bazel
+, bazelTest
+, runLocal
+, unzip
+}:
+
+# Tests that all shebangs are patched appropriately.
+# #!/usr/bin/... should be replaced by Nix store references.
+# #!.../bin/env python should be replaced by Nix store reference to the python interpreter.
```

Comment:
We should even test for `bin/env` if possible, these all indicate bugs if I’m not mistaken.

Reply:
Done, it found one more instance of `#!/usr/bin/env bash` that I've now patched to use `customBash`. I've also pulled the `python` patching underneath `devtools` into the loop that already existed further down in the `postPatch` script.